### PR TITLE
remove-potentially-unused-related-content-route

### PR DESCRIPTION
**Pull request by jonhindle**
### Context

> Does this issue have a Trello card?

https://trello.com/c/UUtVQdiq/266-remove-potentially-unused-related-content-route

> If this is an issue, do we have steps to reproduce?

N/A

### Intent

> What changes are introduced by this PR that correspond to the above card?

code removed

> Would this PR benefit from screenshots?

no

### Considerations

> Is there any additional information that would help when reviewing this PR?

n/a

> Are there any steps required when merging/deploying this PR?

n/a

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/routes/tags.js [hunk 1]
```diff
@@ -40,32 +40,6 @@ const createTagRouter = ({ hubTagsService }) => {
...
-    }
-
-    try {
-      const establishmentId = path(['session', 'establishmentId'], req);
-      const contentType = path(['query', 'contentType'], req);
-
-      const method =
-        contentType === 'series' ? 'relatedSeriesFor' : 'relatedContentFor';
-
-      const data = await hubTagsService[method]({
```
Comment (andrewrlee) [inline server/routes/tags.js:57]: are these methods still used by anything else?

File: server/services/hubTags.js [hunk 2]
```diff
@@ -33,35 +33,8 @@ const createHubTagsService = repository => {
...
-  }
-
-  function relatedContentFor({
-    id,
-    establishmentId,
-    perPage,
-    offset,
-    sortOrder,
-  }) {
-    return repository.relatedContentFor({
```
Comment (andrewrlee) [inline server/services/hubTags.js:52]: is `repository.relatedContentFor` used elsewhere?

Comment (andrewrlee) [reply server/services/hubTags.js:52]: and relatedSeries?

Comment (jonhindle) [reply server/services/hubTags.js:52]: So `repository.relatedContentFor` is used on line 21 still; assuming that route is still valid 🤔 
If you meant `relatedSeriesFor`, it has been completely removed.
I guess you maybe meant `repository.seasonFor`, but that still used on line 11.
To be honest allot of this seems rather unpleasant to unpick; needless complexity 🤷‍♀️

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**